### DISCUSSION TITLE
Import CSV  from MoneyWiz 3

### DISCUSSION THREAD

**Discussion by poudenes** [firefly-iii/firefly-iii#5478] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been discussed before.

### Description

Hi All,

There was a issue names: "Import CSV fail everytime when importing from MoneyWiz 3"
The issue is closed. But I had some questions about this.

I want import my MoneyWiz 3 also into this great application FireFly.

Can someone who imported the information tell me what columns are needed for the import?

### Solution

Someone who can tell me what needed to import this MoneyWiz3 CSV file into FireFly

### What are alternatives?

Not import, but its a whole history.

**Comment by MacPaille** [comment]
Importing a CSV you have to manually select which field is mapped to what value in Firefly. So you can just start an import of some of the transactions and see what you can choose. Always try importing few lines at a time to make sure everything is as you want

**Reply by poudenes** [reply]
When soms field mappings are not correct can I screw up everything? I will backup the database before I try this.

**Reply by JC5** [reply]
Not everything. Only the transactions you import.

**Reply by poudenes** [reply]
So when something is wrong, I can remove the imported transaction and try again with a new line new mappings?

**Reply by MacPaille** [reply]
I would say the best would be taking a backup each time and restoring the backup if a new attempt is needed. Deleted transactions are kept in the DB so you will import multiple times the same ones. And it can have an impact if you use the duplicate finder...

**Reply by poudenes** [reply]
I can't find the restore option in FireFly. Inly a export option.
My database is postgres. So I have to backup that dB and restore that one back when the import fails.

**Reply by MacPaille** [reply]
Yes that's what I meant backup/restore of the DB itself.
